websites/api-docs-smoke-test/src/api-specs/test/api.raml.validation.txt

```diff
@@ -0,0 +1,4 @@
+Model: file:///Users/nkuehn/GitHub/commercetools-docs-kit/websites/api-docs-smoke-test/src/api-specs/test/api.raml
```

Comment:
This should not be included in the merge.

Reply:
I must have messed up something during testing the .gitignores and file deletions.  I'll check, thanks !